### PR TITLE
update chain-validation version

### PR DESCRIPTION
**Pull request by frrist**
go-filecoin causes a panic in the puppet actor here: https://github.com/filecoin-project/specs-actors/blob/master/actors/puppet/puppet.go#L92 -> with error message: "failed to unmarshal nil return" returned here: https://github.com/filecoin-project/go-filecoin/blob/master/internal/pkg/vm/internal/vmcontext/invocation_context.go#L411

### CODE REVIEW

File: internal/pkg/vm/internal/vmcontext/invocation_context.go [hunk 1]
```diff
@@ -408,7 +409,7 @@ func (r returnWrapper) Into(o specsruntime.CBORUnmarshaler) error {
 	// TODO: if inner is also a specsruntime.CBORUnmarshaler, overwrite o with inner.
```
Comment (frrist) [inline internal/pkg/vm/internal/vmcontext/invocation_context.go:409]: Link to PR that last changed this logic: https://github.com/filecoin-project/go-filecoin/pull/3833/files#diff-ac24da57d5301efa8c681bbe720aa2f4R282-R294

File: internal/pkg/vm/internal/vmcontext/invocation_context.go [hunk 2]
```diff
@@ -408,7 +409,7 @@ func (r returnWrapper) Into(o specsruntime.CBORUnmarshaler) error {
...
 	b := bytes.Buffer{}
 	if r.inner == nil {
-		return fmt.Errorf("failed to unmarshal nil return")
+		r.inner = adt.Empty
```
Comment (frrist) [inline internal/pkg/vm/internal/vmcontext/invocation_context.go:412]: without this we get an error here: https://github.com/filecoin-project/specs-actors/blob/master/actors/puppet/puppet.go#L92

File: internal/pkg/vm/internal/vmcontext/testing.go [hunk 3]
```diff
@@ -42,6 +52,21 @@ var _ vstate.VMWrapper = (*ValidationVMWrapper)(nil)
...
+	Add(builtin.InitActorCodeID, &init_.Actor{}).
+	Add(builtin.AccountActorCodeID, &account.Actor{}).
+	Add(builtin.MultisigActorCodeID, &multisig.Actor{}).
+	Add(builtin.PaymentChannelActorCodeID, &paych.Actor{}).
+	Add(builtin.StoragePowerActorCodeID, &power.Actor{}).
+	Add(builtin.StorageMarketActorCodeID, &market.Actor{}).
+	Add(builtin.StorageMinerActorCodeID, &miner.Actor{}).
+	Add(builtin.SystemActorCodeID, &system.Actor{}).
+	Add(builtin.RewardActorCodeID, &reward.Actor{}).
+	Add(builtin.CronActorCodeID, &cron.Actor{}).
```
Comment (anorth) [inline internal/pkg/vm/internal/vmcontext/testing.go:65]: I notice the verified data registry actor is missing here. Would you mind proposing the specs-actors PR to add the list of default actors and their addresses?

File: internal/pkg/vm/internal/vmcontext/invocation_context.go (hunk 2 above)
Comment (anorth) [reply internal/pkg/vm/internal/vmcontext/invocation_context.go]: Thanks. I would like to slightly change how we address this - I will add a commit to this PR.

File: internal/pkg/vm/internal/vmcontext/testing.go (hunk 3 above)
Comment (frrist) [reply internal/pkg/vm/internal/vmcontext/testing.go:65]: Did you mean their _codeIDs_?

Comment (frrist) [reply internal/pkg/vm/internal/vmcontext/testing.go:65]: PR here: https://github.com/filecoin-project/specs-actors/pull/386

**Pull request merged**